include/deal.II/dofs/dof_accessor.h

```diff
@@ -283,6 +283,12 @@ class DoFAccessor : public dealii::internal::DoFAccessorImplementation::
   template <int structdim2, int dim2, int spacedim2>
   DoFAccessor(const InvalidAccessor<structdim2, dim2, spacedim2> &);
 
+  /**
+   * Copy constructor.
+   */
+  DoFAccessor(const DoFAccessor<structdim, dim, spacedim, level_dof_access> &) =
+    default;
```

Comment:
Any reason to implicitly delete the move constructor? I would prefer to just go with the rule of 5.

Reply:
I'd also be in favor to adhere to the rule of five, and additionally specify move constructors and move assignment operators like in #12294.